Source/Core/Common/msvc_sd6_compat.h

```diff
@@ -0,0 +1,226 @@
+// Copyright 2013 Dolphin Emulator Project
```

Comment:
IANAL, I think the date is important since in some country the copyright doesn't last forever. Thus, (ideally) we should write each year where this file was touched. It obviously begins when the file was created, unless that file was moved.